support/ebpf/sched_monitor.ebpf.c

```diff
@@ -33,3 +38,17 @@ int tracepoint__sched_process_free(struct sched_process_free_ctx *ctx)
 exit:
   return 0;
 }
+
+// tracepoint__sched_process_free is a tracepoint attached to the scheduler that frees processes.
+// Every time a processes exits this hook is triggered.
+SEC("tracepoint/sched/sched_process_free/v2")
+int tracepoint__sched_process_free(struct sched_process_free_ctx *ctx)
+{
+  return do_process_free(ctx, ctx->pid);
+}
+
+SEC("tracepoint/sched/sched_process_free/v1")
+int tracepoint__sched_process_free_old(struct sched_process_free_ctx_pre616 *ctx)
```

Comment:
```suggestion
int tracepoint__sched_process_free_pre616(struct sched_process_free_ctx_pre616 *ctx)
```
be consistent with function and context name?